packages/imitation_game/tests/smiley/README.md

```diff
@@ -0,0 +1,11 @@
+# Smiley
+
+This test is an app that just draws one large image to the screen `smiley.png`,
+stretched to fit inside the screen.
+
+The following things are measured:
+
+- 'startupTime' - The time between the start of the process and the rendering of
+  the image to the screen.
+- 'memory' - The amount of system memory the app is using after having rendered
+  the image to the screen.
```

Comment:
while we're here, should we add a cumulative cpu+gpu cost test? (in a different pr as needed)